app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatabaseService.kt

```diff
@@ -219,29 +222,6 @@ class FirestoreDatabaseService internal constructor(
         val imageRef = imagesDir.child(id)
         imageRef.putFile(picture).await()
```

Comment:
Doesn't work if picture is a gs:// url.  I need to either fix or avoid this situation